components/navbars/Nav.tsx

```diff
@@ -1,23 +1,13 @@
 import { AppBar, Box, IconButton, Modal, Toolbar, Typography, useMediaQuery, useTheme } from '@mui/material'
 import * as React from 'react'
-import { useAccount } from 'wagmi'
 import GitHubIcon from '@mui/icons-material/GitHub'
 import ConnectWallet from '../../container/ConnectWallet'
-import useAccountStore from '../../store/accountStore'
 import Link from 'next/link'
 
 export const Nav: React.FC = React.memo(function Nav() {
-  const { address } = useAccount()
-  const setAddress = useAccountStore(state => state.setAddress)
   const theme = useTheme()
   const matches = useMediaQuery(theme.breakpoints.down('sm'))
 
-  React.useEffect(() => {
```

Comment:
moved this over to _app.tsx, so as to make it work for pages that don't have Nav component